documentation/modules/exploit/linux/http/gravcms_exec.md

```diff
@@ -0,0 +1,95 @@
+## Vulnerable Application
+
+### Description
+
+This module exploits arbitrary config write/update vulnerability to achieve remote code execution. Unauthenticated users can execute a terminal command under the context of the web server user.
+
+In versions 1.10.7 and earlier, an unauthenticated user can execute some methods of administrator controller without needing any credentials. Particular method execution will result in arbitrary YAML file creation or content change of existing YAML files on the system.
+
+Technical details about the vulnerability can be found at [here](https://pentest.blog/unexpected-journey-7-gravcms-unauthenticated-arbitrary-yaml-write-update-leads-to-code-execution/) .
+
+### Setup
+
+GravCMS requires only PHP and a web server to run. Please visit following url to find out required php packages.
+
+[https://learn.getgrav.org/17/basics/requirements](https://learn.getgrav.org/17/basics/requirements)
+
+```
+wget https://getgrav.org/download/core/grav-admin/1.7.8
+unzip -x 1.7.8
+chown -R www-data:www-data grav-admin
+```
+
+Once you finished the installation, visit the GravCMS and complete the admin user creation process. To finish up the installation, execute the following command to enable the scheduler.
+
+```
+(crontab -l; echo "* * * * * cd /var/www/html/grav-admin;/usr/bin/php bin/grav scheduler 1>> /dev/null 2>&1") | crontab -
+```
+
+## Verification Steps
+
+Follow [Setup](#setup) and [Scenarios](#scenarios).
```

Comment:
Please, could you use the same format defined in the documentation [template](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md)?